Android/Google.Android.DataTransport/LICENSE.md

```diff
@@ -0,0 +1,29 @@
+**Xamarin is not responsible for, nor does it grant any licenses to, third-party packages. 
+Some packages may require or install dependencies which are governed by additional 
+licenses.**
+
+Note: This component depends on [com.google.android.datatransport](https://mvnrepository.com/artifact/com.google.android.datatransport)), 
+which is subject to the [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+### Xamarin Component for com.google.android.datatransport for Xamarin.Android
+
+**The MIT License (MIT)**
+
+Copyright (c) .NET Foundation Contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software 
+and associated documentation files (the "Software"), to deal in the Software without restriction, 
+including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so,
+ subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial 
+portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT 
+NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND 
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES 
+OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN 
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+20191023
```

Comment:
Don’t know if this date should be updated

Reply:
spot on.
THanks